internal/guest/storage/devicemapper/targets.go

```diff
@@ -80,10 +88,17 @@ func CreateVerityTarget(ctx context.Context, devPath, devName string, verityInfo
 		trace.Int64Attribute("sectorSize", dmBlocks),
 		trace.StringAttribute("verityTable", verityTarget.Params))
 
-	mapperPath, err := CreateDevice(devName, CreateReadOnly, []Target{verityTarget})
+	devMapperPath, err := CreateDeviceWithRetryErrors(
+		ctx,
+		devName,
+		CreateReadOnly,
+		[]Target{verityTarget},
+		unix.ENOENT,
+		unix.ENXIO,
+	)
 	if err != nil {
-		return "", errors.Wrapf(err, "failed to create dm-verity target. device=%s", devPath)
+		return "", fmt.Errorf("frailed to create dm-verity target for device=%s: %w", devPath, err)
```

Comment:
Is this specific to container layer devices?  If so, would it be possible to just make an error message that says something like "container layer for container [X] could not be mounted in time.  Retrying may resolve this issue.".  This way the person consuming the error knows a little bit more specifically what happened and how to move forward?

Reply:
not necessarily, at this point GCS isn't aware if it's a container layer being mounted or not. The host, however, is. The host side error message will have something like: `failed to add LCOW layer: failed to add SCSI layer: failed to modify UVM with new SCSI mount: guest modify: guest RPC failure: frailed to create dm-verity target for device=/dev/sda: device-mapper table load: no such device: unknown`, so I don't we need to word this differently.